tools/qtc_templates/kuesa/project/empty_quick/main.qml

```diff
@@ -0,0 +1,67 @@
+import Qt3D.Core %{Qt3DCoreVersion}
+import Qt3D.Render %{Qt3DVersion}
+import Qt3D.Input %{Qt3DInputVersion}
+import Qt3D.Extras %{Qt3DVersion}
+import QtQuick %{QtQuickVersion}
+import QtQuick.Scene3D %{Scene3DVersion}
+import Kuesa %{KuesaVersion} as Kuesa
+
+Item {
+    id: root
+
+    Scene3D {
+        id: scene3d
+        anchors.fill: parent
+        aspects: ["render", "input", "logic", "animation"]
+
+        Kuesa.SceneEntity {
+            id: root3D
+
+            components: [
+                RenderSettings {
+                    activeFrameGraph: Kuesa.ForwardRenderer {
+                        id: frameGraph
+                        camera: root3D.cameras.names.length ? root3D.cameras.find(root3D.cameras.names[0]) : mainCamera
+                        clearColor: "white"
+                    }
+                },
+                InputSettings { },
+                DirectionalLight {
+                    worldDirection: frameGraph.camera.viewVector
+                    intensity: 0.8
+                    color: "white"
+                }
+            ]
+
+            Camera {
+                id: mainCamera
```

Comment:
I'd rename that to "fallbackCamera" since it's mostly just in case the export wouldn't provide something, isn't it?

Reply:
depends if you are using the GLTFLoader (by specifying a path in the wizard) or not I suppose, this would also have to be addressed in the other template.